src/style.css

```
@@ -1 +1,71 @@
-/* Styles go here */
+
+
+* {
+  box-sizing: border-box;
+}
+html {
+  font-family: Roboto,Helvetica,sans-serif
+}
+body {
```

you should separate different rules by empty space line
it will be more clear to read
```suggestion
}

body {
```